### ISSUE TITLE
Frederick Establishment Sites - 2/11/16

### ISSUE THREAD

**Issue by nbyers1** [kerrymcmahon/Frederick-County-Stormwater#8]
Frederick Conflicts (Contract advertises 4/19):

1.For the SWM In Design Concept and Under Contract conflicts, I suspect it would be useful to host another meeting between Stacey & the SWM PM to determine the best course of action for each site (reshaping/splitting/deleting). This contract still has a decent amount of time for us to work out changes, although we would like to address this sooner than later. In FR, these are:
- FR-MP-1007 in conflict with 100170UT
- FR-KSL-1001 in conflict with 100044UT
- FR-KSL-1002 in conflict with 100044UT
1.  For the  sites whose task name mentions Tree Review – do we need to worry about these sites? They are still Status = RR. In FR, these are:
- FR-SRF-0010 in conflict with 100171UT
1.  There are 44 more conflicts in FR; however, these conflicts are with non-fast tracked sites that are still at ‘Recommended for Restoration’ or ‘On Hold’ status & ‘GIS Desktop Assessment’ source, so I believe there is no real conflict at present.

[SWM_Conflicts_FR_WA.xlsx](https://github.com/kerrymcmahon/Frederick-County-Stormwater/files/126625/SWM_Conflicts_FR_WA.xlsx)

**Comment by mparlakian** [comment]
FR-KSL-1001 – In concept design.  Put tree site on hold until LOD is established.  There will be remaining area for tree establishment.  
FR-KSL-1002 – In concept design.  Put tree site on hold until LOD is established.  There will be remaining area for tree establishment.
FR-MP-1007 - In concept design.  Put tree site on hold until LOD is established.  There will be remaining area for tree establishment.
FR-SRF-0010 – This SWM site may be put on hold.  Imp. area is low.
FRGr2-JMT-1434 – This SWM site may be put on hold.  Imp. area is low.
FRGr2-JMT-1501 – Put tree site on hold.  Potential high imp. site.
FRGr2-JMT-1502 – Put tree site on hold.  Potential high imp. site.
FRGr2-JMT-1469 – Tree shape may be reshaped to avoid channel area.  Potential high imp. site.
FRGr2-JMT-2551 – SWM site will most likely not impact small corner of tree site on abutment ground slope.  SWM site may be adjusted to avoid tree site.
FRGr2-JMT-1948 – This SWM site may be put on hold.  Imp. area is low.
FRGr2-JMT-1949 - This SWM site may be put on hold.  Imp. area is low.
FRGr2-JMT-1950 – Can tree site be reshaped to avoid channel area adjacent to roadway?
FRGr2-JMT-1959 – This SWM site may be put on hold.  Appears too steep.
FRGr2-JMT-1962 – This SWM site may be put on hold.  Appears steep and low imp. area.
FRGr2-JMT-1963 – This SWM site may be put on hold.  Side slope appears too steep for grading for most of its distance.
FRGr2-JMT-1964 – This SWM site may be put on hold.  Low imp. area.
FRGr2-JMT-2042 - This SWM site may be put on hold.  Side slope appears too steep for grading for most of its distance.
FRGr2-JMT-1931 - This SWM site may be put on hold.  Side slope appears too steep for grading.
FRGr2-JMT-2135 - This SWM site may be put on hold.  Side slope appears too steep for grading.
FRGr2-JMT-2120 – Reshape tree site slightly to avoid overlap with SWM.
FRGr2-JMT-2121 - This SWM site may be put on hold.  Low imp. area.
FRGr2-JMT-1972 – Reshape tree site to a void small overlap of channel BMP.
FRGr2-JMT-1969 - Reshape tree site to a void small overlap of channel BMP.
FRGr2-JMT-2364 – SWM site will most likely not occupy entire shape shown.  Difficult to tell without performing grading.  There will still be space available for planting after BMP is designed.  Can tree site be put on hold?
FRGr2-JMT-2360 – Pot. imp. may be small even with offsite.  Significant excavation unless only a linear facility pursued.  Truncate tree area to only the flat top of slope. 
FRGr2-JMT-2361 - This SWM site may be put on hold.  Side slope appears too steep for grading for most of its distance.
FRGr2-JMT-2362 – SWM can probably be reshaped to avoid small conflict.
FRGr2-JMT-0206 - This SWM site may be put on hold.  Low imp. area.
FRGr2-JMT-0184 - This SWM site may be put on hold.  Low imp. area.  I-70 superelevated away.  Offsite imp. area contributes.
FRGr2-JMT-0185 – This SWM may be put on hold.  Receives sig. Offsite, but also a lot of imp. from WB I-70.  Not enough flat space for BMP.
FRGr2-JMT-1991 – This SWM site may be put on hold.  Low imp. area.
FRGr2-JMT-2054 - This SWM site may be put on hold.  Side slope appears too steep for grading for most of its distance.
FRGr2-JMT-1953 – Swale site may be possible near toe of slope for 0.65 ac. imp. area.  Please place tree site on hold.
FRGr2-JMT-1954 - Swale site may be possible near toe of slope.  Please place tree site on hold.
FRGr2-JMT-1955 - This SWM site may be put on hold.  Side slope appears too steep for grading.
FRGr2-JMT-2554 – Truncate tree establishment site to avoid BMP footprint.
FRGr2-JMT-2555 – Measured imp. area appears less than that report at DE.  Steep slope/significant grading.  SWM site may be put on hold.
FRGr2-JMT-2552 – Retain northern polygon of tree establishment site.  BMP may be viable at southern polygon.
FRGr2-JMT-1462 – Place tree establishment site on hold.  Adjacent FR-MPO-1007 is being designed.  Depending on final outcome, final shape of tree establishment site can be made.
FRGr2-JMT-1471 – Final DA for adj. FR-MP-1007 is being established in concept design.  Once complete, this site may eventually be removed because the area is captured by FR-MP-1007.
FRGr2-JMT-1960 - This SWM site may be put on hold.  Side slope appears too steep for grading.
FRGr2-JMT-1961 - This SWM site may be put on hold.  Side slope appears too steep for grading.
FRGr2-JMT-1962 - This SWM site may be put on hold.  Low imp. area.
FRGr2-JMT-2556 – Reshape tree site to remain off of preliminary swale shape.
FRGr2-JMT-1434 – Reshape tree site to remain off of preliminary swale shape.
FRGr2-JMT-1208 – Keep SWM site.  It will most likely change shape to be multiple cells of BMP footprints, leaving area for tree establishment.
FRGr2-JMT-1028 – Keep SWM site.  Receives additional imp. area via the median inlet.  If median facility is constructed, This site may be less desirable.
FRGr2-JMT-1461 – Place tree site on hold.  Appears to drain to FR-MP-1007, but need to confirm.  Once concept design is complete for FR-MP-1007, 1461 may be removed if within DA. 
FRGr2-JMT-1510 - Place tree site on hold.  Appears to drain to FR-MP-1007, but need to confirm.  Once concept design is complete for FR-MP-1007, 1461 may be removed if within DA.

**Comment by mparlakian** [comment]
Natalie,  

For these three sites, I am giving guidance to the consultant working on thee sites to continue the concept design.  Their designs may impact installed trees.  Once concept is complete, new footprints and LODs will be available to see how the tree site is affected.

FR-KSL-1001
FR-KSL-1002
FR-MP-1007

**Comment by nbyers1** [comment]
Matt,

Thanks for the input. We will put tree sites 100044UT (in conflict with FR-KSL-1001 & 1002), 100137UT (in potential conflict with FR-MP-1009), and 100170UT (in conflict with FR-MP-1007) as 'crew day' sites in our D7 establishment contract, which means we can be flexible about changing their shape or pulling them out part way through. Please let us know when the LODs are finalized and we will adjust our shapes accordingly and notify the contractor. 

If any of the other SWM sites are considered to move forward, a new conflict should be opened at that time.

Thanks!
Natalie

**Comment by nbyers1** [comment]
Also, if you have any info regarding when LODs will be available and when the SWM sites are anticipated to begin construction, that would be helpful.

**Comment by kerrymcmahon** [comment]
The LOD delivery should return by the end of March hopefully. I will update accordingly making reference to these features.

**Comment by kerrymcmahon** [comment]
I have placed a few of the SWM sites on hold. The few that we are keeping will be left in the database, and conflicts will be address at the time they move ahead past field. NO tree sites should be placed on hold except the three discussed that conflict with KSL 1001, 1007, 1002

I will reopen conflict 00209 which included 2 of three of these sites, and we;ll keep it open until the LODs are loaded.

Natalie: What, if any, changes should I make in the database for LUCHANGE?

**Comment by nbyers1** [comment]
Kerry, I believe no changes are needed in the DB at this time. We will revisit this once the LODs are received.
